.gitignore

```diff
@@ -1,3 +1,5 @@
+secret*
```

Comment:
Great job! 
It is good to make sure none of the secrets and tokens are committed to git